app/views/welcome/index.html.erb

```diff
@@ -384,29 +384,46 @@
     </div>
   </div>
 
-  <div class="unit light partners" id="partners">
-    <div class="row">
-      <div class="col-md-12">
-        <h2 class='sectionHead dark'>
-          Our Awesome Partners
-        </h2>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-3 col-sm-6">
-        <%= link_to image_tag("partners/briebug_solutions.png"), "http://briebugsoftwaresolutions.com/" %>
-      </div>
-      <div class="col-md-3 col-sm-6">
-        <%= link_to image_tag("partners/radial-logo.png"), "http://radialdevgroup.com/" %>
-      </div>
-      <div class="col-md-3 col-sm-6">
-        <%= link_to image_tag("partners/venture_for_america.png"), "http://ventureforamerica.org/" %>
+  <!-- hackathon code -->
+    <div class="unit light tips" id="tips">
+      <div class="row">
+        <div class="col-md-12">
+          <h2 class='sectionHead dark' style="padding: 0;">
+            Tips to Get Started
+          </h2>
+        </div>
       </div>
-      <div class="col-md-3 col-sm-6">
-        <%= link_to image_tag("partners/girl_develop_it.png"), "https://www.girldevelopit.com/" %>
+      <div class="row" style="margin-left:50px">
+        <div class="col-md-4 tipsinfo">
+          <h3>Peers</h3>
+          <div id="peers-tips">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+          quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+          consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+          cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+          proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+        </div>
+        <div class="col-md-4">
+          <h3>Mentors</h3>
+          <div id="mentors-tips">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+          quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+          consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+          cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+          proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+        </div>
```

Comment:
Change to: 
```
`          <h3>Mentors</h3>
          <p> Thank you for volunteering your time to give back to those coming up behind you on the ladder! Here's some ways to make the most of it: </p>
          <ul>
            <li><b>Start by asking questions.</b> It's important to understand where someone is in their career and where they want to go. Once you know that, it's just a matter of recognizing the roadblocks they face, and how to get past them.
            </li>
            <li> <b> Keep listening. </b> Often your mentee has their own answer. Strongly affirm that answer when they find it. The conclusions they come to themselves stick better than anything you could say. 
            </li>
            <li> <b> Be genuine. </b> If it's not a fit after the first phone call, it's ok to move on and get a new mentee next month. This is about chemistry! Let them find someone who does believe in them to save both of you a lot of head and heart ache.
            </li>
            <li> <b> Set expectations.</b> If you really believe in your mentee and want to keep in contact, agree on how you will communicate, when and what your goals are. This will help a great deal in paving the way to good mentor/mentee relationship.
            </li>
          </ul> `
```